app.rb

```diff
@@ -0,0 +1,65 @@
+require './student'
+require './teacher'
+require './book'
+require './classroom'
+
+class App
+  attr_reader :books, :classrooms, :students, :teachers
+
+  def initialize
```

Comment:
You made a great effort to create this project. However, some changes need to be done.